docs/index.rst

```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
-The *Micro XRCE-DDS Agent* acts on behalf of the *Micro XRCE-DDS Clients* and enables them to take part as DDS publishers and/or subscribers in the DDS Global Data Space.
+*Micro XRCE-DDS Agent* (server) makes possible this communication acting on behalf of the *Micro XRCE-DDS Clients*
+and enables them to take part as DDS publishers and/or subscribers in the DDS Global Data Space.
 
 *Micro XRCE-DDS* provides both, a plug and play *Micro XRCE-DDS Agent* and an API layer which allows you to implement your *Micro XRCE-DDS Clients*.
 
 .. image:: images/xrcedds_architecture.svg
 
+Main features
+~~~~~~~~~~~~~
+High performance.
+    Uses a static low level serialization library `(MicroCDR) <https://github.com/eProsima/Micro-CDR>`_ that serialize in `XCDR <https://www.omg.org/spec/DDS-XTypes/1.2/PDF>`_.
+
+Low resouces.
+    The client library is dynamic memory free and static memory free.
+    This means that the only memory charge is due by the stack growth.
+    It can manage a simple publisher/subscriber with less of 2KB of RAM.
+    Besides, the client is built with a *profiles* concept, that allows to add or remove functionality to the library in order to reduce the binary size.
+
+Multi-platform.
+    The OS dependencies are treated as pluggable modules.
+    The user can implement easily his own platform modules in order to use *Micro XRCE-DDS Client* library in his specific platform.
+    By default the proyect can run over `Linux`, `Windows` and `Nuttx`.
+
+Free and Open Source.
+   The client library, among with the agent executable, the generator tool and internal dependencies as *MicroCDR* or *FastRTPS* are all of them free and open source.
+
+Easy to use.
+    The project comes with examples of a publisher and a subscriber,
+    an example of how MicroXRCE-DDS is deployed into a stage and
+    an interactive demo that can be used with the `ShapesDemo <https://github.com/eProsima/ShapesDemo>`_ with the purpose of understanding the DDS-XRCE protocol and making tests.
+    The client API es completely explained and documentation comes with a guided example of how create your own client application.
+
+Implementation of the DDS-XRCE standard.
+    `DDS-XRCE <https://www.omg.org/spec/DDS-XRCE/1.0/Beta1/PDF>`_ is a standard communication protocol of OMG consortium
+    focused in communicating eXtremely Resource Constrained Environments with the DDS world.
+
+Best effort and reliable communication.
+    *Micro XRCE-DDS* supports both, *best effort* for a fast and light communication and *reliable* when the communication reliability is needed.
+
+Transport layer pluggable.
+    Micro XRCE-DDS is not built over a specific transport as *serial* or *UDP*.
+    It is agnostic about the transport used, and give to the user the posibility of implement easily his own transport.
+    By the fault, *UPD*, *TCP*, and *serial* transports are provided.
+
+Commercial Support.
+    Available: `info@eprosima.com`
```

Comment:
```suggestion
```

Reply:
I need this line, because the rst style request me a new line over the "feature title"

Reply:
Need to change the email to point to `support@eprosima.com` then.